Functions.Templates/Bindings/bindings-bundle-v4.json

```diff
@@ -1859,6 +1859,116 @@
           "help": "$signalROut_connectionString_help"
         }
       ]
+    },
+    {
+      "type": "sql",
+      "displayName": "$sqlIn_displayName",
+      "direction": "in",
+      "enabledInTryMode": false,
+      "documentation": "$content=Documentation\\sqlIn.md",
+      "extension": {
+        "id": "Microsoft.Azure.WebJobs.Extensions.Sql", "version": "1.0.122-preview"
+      },
+      "settings": [
+        {
+          "name": "name",
+          "value": "string",
+          "defaultValue": "object",
+          "required": true,
+          "label": "$sqlIn_name_label",
+          "help": "$sqlIn_name_help",
+          "validators": [
+            {
+              "expression": "^[a-zA-Z][a-zA-Z0-9]{0,127}$",
+              "errorText": "[variables('parameterName')]"
+            }
+          ]
+        },
+        {
+          "name": "commandText",
+          "value": "string",
+          "defaultValue": "Select * from object",
+          "required": true,
+          "label": "$sqlIn_commandText_label",
+          "help": "$sqlIn_commandText_help"
+        },
+        {
+          "name": "commandType",
+          "value": "string",
+          "defaultValue": "Text",
+          "required": false,
+          "label": "$sqlIn_commandType_label",
+          "help": "$sqlIn_commandType_help"
+        },
+        {
+          "name": "connectionStringSetting",
+          "value": "string",
+          "defaultValue": "SqlConnectionString",
+          "required": true,
+          "label": "$sql_connection_label",
+          "help": "$sql_connection_help",
+          "placeholder": "[variables('selectConnection')]"

```

Comment:
What does this evaluate to?

Reply:
https://github.com/Azure/azure-functions-templates/blob/b9c124c468bed78261f195f7fe0ddb8b74458a21/Functions.Templates/Resources/Resources.resx#L255

Reply:
Huh, wonder why the syntax is different here. @soninaren - I would expect it to be `$variables_selectConnection`, I guess this is a special field for some reason?